modules/libcom/src/log/logClient.c

```diff
@@ -537,7 +568,7 @@ void epicsStdCall logClientShow (logClientId id, unsigned level)
             pClient->connectCount);
     }
     if (level>1) {
-        printf ("log client: %u bytes in buffer\n", pClient->nextMsgIndex);
+        printf ("log client: %zu bytes in buffer\n", pClient->nextMsgIndex);
```

Comment:
We've been removing the `%z` (`size_t`) prefix from our `printf()` formats as it isn't (wasn't?) portable. I forget which targets don't/didn't handle it, I'm pretty sure Windows was one of them though; have all those targets now been dropped? That's why the existing code uses `unsigned`, which is fine for log message lengths which could never be as large as 2**31 bytes.

Reply:
Alternatively, I can cast to `long long` and use `%llu` here.